images/deno/configs/latest.apko.yaml

```diff
@@ -19,10 +19,11 @@ paths:
 work-dir: /app
 
 environment:
-  PATH: /usr/sbin:/sbin:/usr/bin:/bin
```

Comment:
Should we include `/sbin` in the PATH too?

Reply:
I think the answer will be **yes**, because a couple packages install themselves there: https://github.com/search?q=repo%3Awolfi-dev%2Fos%20%24%7B%7Btargets.subpkgdir%7D%7D%2Fsbin&type=code